modules/unmanaged-operators.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * architecture/architecture-installation.adoc
+// * updating/updating-cluster-between-minor.adoc
+
+[id="unmanaged-operators_{context}"]
+= Unmanaged Operators
```

Comment:
Was trying to get the word "support" in the title, like simply "Unmanaged Operators support", though at a glance that makes it look like it's supported. "Supportability" is awkward, as is "non-support". And "unsupport" would make it two "un-" words in a single title...

Reply:
Support for unmanaged operators.
Support policy for unmanaged operators.

Reply:
I'd be tempted to call it "Operator management states" and add a paragraph to the update topic that says that you need to care about them before you update.

It also feels like you might be able to extract a procedure from this concept, but I don't know how many people have disabled their CVO management.

Reply:
> It also feels like you might be able to extract a procedure from this concept, but I don't know how many people have disabled their CVO management.

Definitely could, mostly wondering about should.

Reply:
> > It also feels like you might be able to extract a procedure from this concept, but I don't know how many people have disabled their CVO management.
> 
> Definitely could, mostly wondering about should.

I don't have the data for that, but if it's something that you expect people to need to do instead of just data you want them to have, it should be a procedure.

Reply:
Yeah, that's the question. Waiting for support folks opinion. @sferich888 ?